### ISSUE TITLE
ARCH environment Setup

### ISSUE THREAD

**Issue by tmiles2** [emory-libraries/dlp-curate#560]
3 SOLR, Curate, Lux, Cantaloupe, Fedora

- [x] Add Arch env to ansible scripts
- [x] Request DNS for arch environments
- [x] Add new rules to f5 and internal Elastic Load Balancer for arch envs
- [x] Request new fedora user from AWS@emory team for new arch fedora bucket

**Comment by tmiles2** [comment]
@SolomonHD I need an update on this work when you get a chance.

**Comment by SolomonHD** [comment]
I'm still waiting on the fedora-arch user to be created. I contacted Paul Peterson and hopefully he'll get it to me today.

**Comment by SolomonHD** [comment]
I received the fedora-arch user credentials a few days ago and Gerry updated the shibboleth metadata for curate-arch. I will proceed with building out the environments today.

**Comment by SolomonHD** [comment]
I've got the arch environment up at curate-arch.library.emory.edu. Currently it's giving an invalid SSL name, not a big deal but I'll double check the cert on it.

@rotated8 I made changes to capistrano. I created another role for the collection steps called :collection and create a minimal deploy namespace for each environment that does not include the collection step. This will give us a quick way to bypass the collection steps in case they are not working. I'm going to release a PR for this
